### PR TITLE
Allows the use of relative addresses to load vizjsons

### PR DESCRIPTION
**Pull request by fdansv**
@alonsogarciapablo what do you think?

Fixes https://github.com/CartoDB/cartodb.js/issues/519

### CODE REVIEW

**Comment by fdansv** [comment]
ah crap need to change a couple things

File: src/core/loader.js [hunk 1]
```diff
@@ -30,6 +34,12 @@ var Loader = cdb.vis.Loader = cdb.core.Loader = {
     }
   },
 
+  getNoJSONP: function(url, callback) {
+    $.get(url).done(function(data){
```
Comment (javisantana) [inline src/core/loader.js]: no jquery is allowed here, it could not be loaded

**Comment by javisantana** [comment]
hey, this thing is as easy as remove this check https://github.com/CartoDB/cartodb.js/blob/develop/src/api/layers.js#L45

**Comment by fdansv** [comment]
damn, it seems i've overcomplicated it a bit... ok changing

File: test/spec/api/layers.spec.js [hunk 2]
```diff
@@ -222,6 +222,17 @@ describe('api.layers', function() {
 
       });
 
+      it("should load local viz.json correctly", function(done) {
+        sinon.spy($, "get");
+        cartodb.createLayer(map, "../../demos/viz.json")
+            .addTo(map);
+        setTimeout(function(){
+          expect($.get.getCall(0)); // If jquery get has been called, means it's loading it as a local, non jsonp vizjson
```
Comment (alonsogarciapablo) [inline test/spec/api/layers.spec.js]: Did you try `expect($.get).toHaveBeenCalled()`?

Comment (fdansv) [inline test/spec/api/layers.spec.js]: I was using sinon, not jasmine in this case. I'm closing this PR and opening a new one with the more simple change :)

**Pull request closed without merging**